armoryengine/WalletUtils.py

```diff
@@ -315,6 +315,32 @@ def detectHighestUsedIndex(self):
       for _, wlt in self._walletMap.items():
          wlt.detectHighestUsedIndex()
 
+################################################################################
+## Wallet Data Classes
+################################################################################
+
+class WalletEntry(object):
+   """Represents a single wallet entry with all its properties."""
+
+   def __init__(self, walletId, path, filename, displayName, staged, 
```

Comment:
Why the boilerplate when there's an entire method preparing the data elsewhere?

This should be the body of `_createWalletEntry`